contracts/core/RebalancingSetTokenFactory.sol

```diff
@@ -110,16 +110,25 @@ contract RebalancingSetTokenFactory {
         returns (address)
     {
         // Expecting caller to be Core
-        require(msg.sender == core, "ONLY_CORE_CAN_CREATE_REBAL_SET");
+        require(
+            msg.sender == core,
+            "RebalancingSetTokenFactory.create: Unauthorized sender"
+        );
 
         // Ensure components and units length are 1
-        require(_components.length == 1 && _units.length == 1, "ARRAY_LENGTHS_MUST_BE_ONE");
+        require(
+            _components.length == 1 && _units.length == 1,
+            "RebalancingSetTokenFactory.create: Invalid array lengths"
```

Comment:
Maybe split this one into 2 `require`s and then specify in the error which array length is the bad one?